src/types.db

```diff
@@ -68,6 +68,7 @@ email_size		value:GAUGE:0:U
 entropy			value:GAUGE:0:4294967295
 expired_keys    value:GAUGE:0:U
 fanspeed		value:GAUGE:0:U
+fh			value:GAUGE:0:U
```

Comment:
I'm not very happy about this name. Could we either (re)use "files" for this plugin or rename this type to "handles"?